src/Umbraco.Core/Strings/DefaultShortStringHelper.cs

```diff
@@ -532,7 +533,7 @@ internal void CopyTerm(string input, int ipos, char[] output, ref int opos, int
                     {
                         s = term.Substring(ipos, 2);
```

Comment:
Substring creates a new string. Consider defining s as a Span<char>/ Memory<char> and use slice instead of substring.